frontend/src/components/common/common.ts

```diff
@@ -20,4 +20,6 @@ export { default as CreateCommentForm } from './create-comment-form/create-comme
 export { default as PodcastList } from './podcasts-list/podcasts-list';
 export { default as PodcastItem } from './podcast-item/podcast-item';
 export { default as Player } from './player/player';
+export { MaskedInput } from './masked-input/masked-input';
```

Comment:
```suggestion
export { default as MaskedInput } from 'react-maskedinput';
```
don't forget about import order